ksqldb-udf-quickstart/pom.xml

```diff
@@ -64,7 +64,8 @@
                 <artifactId>maven-archetype-plugin</artifactId>
                 <version>2.2</version>
                 <configuration>
-                    <skip>${skipTests}</skip>
+                    <!-- Skipping this integration test because it doesn't work with nano versioning. -->
+                    <skip>true</skip>
```

Comment:
We shouldn't leave this disabled, as it would mean the archetype could be broken, and we'd never know until someone complained.

Reply:
https://github.com/confluentinc/ksql/issues/6480